compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -435,32 +435,79 @@ static const char *opCodeToNameMap[] =
    "rev16w",
    "rev16x",
    "rev32",
-   "fmov_stow",
-   "fmov_wtos",
-   "fmov_dtox",
-   "fmov_xtod",
-   "fcvt_stod",
-   "fcvt_dtos",
-   "fcvtzs_stow",
-   "fcvtzs_dtow",
-   "fcvtzs_stox",
-   "fcvtzs_dtox",
-   "scvtf_wtos",
-   "scvtf_wtod",
-   "scvtf_xtos",
-   "scvtf_xtod",
    "fmovimms",
    "fmovimmd",
    "fcmps",
    "fcmps_zero",
    "fcmpd",
    "fcmpd_zero",
    "fmovs",
-   "fmovd",
-   "fabss",
-   "fabsd",
-   "fnegs",
-   "fnegd",
+	"fabss",
+	"fnegs",
```

Comment:
Indentation of lines from "fabss" to "fcvt_htod" is different from that of other lines.